File: level20/classwork/task2.py
#```2) მომხმარებელს შემოატანინეთ რიცხვი, შემდეგ თუ ეს რიცხვი დადებითია დაბეჭდეთ ეს და კიდევ შეამოწმეთ ლუწია თუ კენტი, ხოლო თუ არაა დადებითი მხოლოდ დაბეჭდეთ რომ უარყოფითია```
number=input("enter a number")
if number >0:
    print("number is positive")
    if number %2==0:
        print("even")
    else:
        print("odd")
else:
    print("number is negative")

File: level71/classwork/task2.js
function NumberIn(number) {
    this.number = number;
    this.even = number % 2 === 0;
}

let numbers = [];

for (let i = 1; i <= 10; i++) {
    let obj = new NumberIn(i);
    numbers.push(obj);
}

console.log("Numbers array:", numbers);
